package.json

```diff
@@ -1,7 +1,6 @@
 {
   "private": true,
   "scripts": {
-    "install": "yarn build",
```

Comment:
With this change, what do you think about adding `yarn build` to our `post-merge` hook?

https://github.com/aws-amplify/amplify-ui/blob/5e575bdf522eacd0a5cfa0c32be6da52246e8984/.husky/post-merge#L5-L6

When switching branches, it can be risky to not rebuild the packages. (For example, if you're working with `yarn react dev`, but forget to `yarn ui build` after switching)

Reply:
That's a good catch, let's do that.